projects/packages/waf/rules/rules.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/** @var $waf \Automattic\Jetpack\Waf\WafRuntime */
+$rule = (object) array( 'id' => 1337, 'reason' => '', 'tags' => array (
+	0 => 'stuff',
+) );
+if($waf->match_targets(array (
+),array (
+	'query_string' =>
+		array (
+		),
+),'rx','#.*jetpack-waf-always-block.*#',false,true)) {
+	$rule->reason = 'Intentionally blocking this test-request';
+	return $waf->block('block',$rule->id,$rule->reason,403);
+}
```

Comment:
@bhoop Do what exactly now? Remove it? I'd prefer to have the pseudo-blocking rule in there in the hopes that we will also transfer it (the rule) once we make the step to have the rules generated. From what I understand, we will commit the rules in a first step too, so I'd think it could stay it this place for now - or did I get that wrong? 🤔 

For when it is being generated (later), where would it go? I would have expected that we just do not have it in VCS anymore and the file then gets overwritten by the plugin every time it updates the rules?

Reply:
I am reworking the pseudo-blocking ruleset to have rules that trigger all the currently supported (or the important) Directives, Variables, Transformations, Operators and Actions.
Not sure if compile them to a PHP file on the plugin is needed, I setup scripts that will do the conversion on our side.